FlowMagic-SDKTests/ScreenFlowTests/MockScreenProvider.swift

```diff
@@ -0,0 +1,75 @@
+////
+////  MockScreenProvider.swift
+////  FlowMagic-SDK
+////
+////  Created by Kedar Dhere on 3/25/23.
+////
+//
```

Comment:
Nitpick:

Was this generated by your template?  Conventionally the file header only has `//` in front, not `////` (four slashes), especially because lines that start with `///` (three slashes) are docstrings and this is not a docstring I don't think.